pyuvdata/utils.py

```diff
@@ -3783,7 +3789,13 @@ def calc_sidereal_coords(
 
 
 def get_lst_for_time(
-    jd_array, latitude, longitude, altitude, astrometry_library=None, frame="itrs"
+    jd_array=None,
+    latitude=None,
+    longitude=None,
+    altitude=None,
+    astrometry_library=None,
+    frame="itrs",
+    telescope_loc=None,
```

Comment:
Can you add this to the docstring? It looks like it can be a tuple of lat/lon/alt or an EarthLocation or a MoonLocation?

Reply:
Added.